edc-controlplane/edc-controlplane-base/build.gradle.kts

```diff
@@ -77,3 +77,34 @@ dependencies {
     runtimeOnly(libs.edc.controlplane.callback.dispatcher.http)
 
 }
+
+tasks.register("downloadOpenapi") {
```

Comment:
this should probably be in the root build file, because the dataplane also has an API and thus openAPI files

Reply:
I was thinking, do we need to list the dataplane api as well? Because those should be considered internal ones, same as "control" and "dsp", that's why I'm working to filter them out (update to PR will come up soon).
likely to external viewers only "management" and "observability" would be interesting, wdyt?

otherwise a possibility could be to publish them separately as we're doing on upstream

Reply:
not "external", but "internal". 
token refresh is already part of the merged spec because it's not downloaded from maven repo but generated by the `resolve` task.

anyway I will add the task for both control-plane and data-plane, for `management-api`, `observability-api` and `public-api`, then let's wait for some feedback

Reply:
sorry, i meant to say that I would not consider them _internal_ ofc. 

> token refresh is already part of the merged spec because it's not downloaded from maven repo but generated by the resolve task.

i know, my point was that the dataplane very much has APIs that are public, such as DPS and token refresh. especially with DPS i see no reason _not_ to include it.